tests/sdk/test_image_construction.py

```diff
@@ -0,0 +1,51 @@
+from tests.base import TestBase
+
+from aim.sdk import Image
+import numpy as np
+
+
+class TestImageConstruction(TestBase):
+        # greyscale, dims = (1,2,2)
+    img1 = np.array([[[0, 1], [1, 0]]], np.uint8)
+    # greyscale, dims = (2,2)
+    img2 = np.array([[0, 1], [1, 0]], np.uint8)
+    # RGB, dims = (3,2,2)
+    img3 = np.array([[[0, 1], [1, 0]], [[0, 1], [1, 0]], [[0, 1], [1, 0]]], np.uint8)
+
+    def test_image_from_torch_tensor(self):
+        try:
+            import torch
+        except (ImportError, AssertionError):
+            raise ValueError('Cannot convert from torch.Tensor')
```

Comment:
so this tests are failing if tensorflow/pytorch are not installed. need to add both in `requirements.dev.txt`

Reply:
done